### ISSUE TITLE
How to set up cosine learning rate to decrease to 0 at the end of training.

### ISSUE THREAD

**Issue by lejekjr** [google/automl#1122]
Version automl/efficientdet/keras/  tag: 1.2

I have two trainings one after another.
First training:
`python /repo/automl-1.2/efficientdet/keras/train.py \
    --train_file_pattern=s3://wsum/kubeflow/DrOetker_BE_automl/Run_of_AutoML_train_2021-11-24-12:06:30_1d50c/2021-11-24T15:05:2400:00/data/tfrecords/train.tfrecord \
    --val_file_pattern=s3://wsum/kubeflow/DrOetker_BE_automl/Run_of_AutoML_train_2021-11-24-12:06:30_1d50c/2021-11-24T15:05:2400:00/data/tfrecords/test.tfrecord \
    --model_name=efficientdet-d2 \
    --model_dir=s3://wsum/kubeflow/DrOetker_BE_automl/Run_of_AutoML_train_2021-11-24-12:06:30_1d50c/2021-11-24T15:05:2400:00/model_dir \
    --pretrained_ckpt=s3://wsum/lunalewa/WSUM-4347/DrOetker_BE_automl_fine_tune_WSUM-4739_train_2021-10_20_08_57_45_c9141_dd_24112021/chckpt_39 \
    --batch_size=5 \
    --eval_samples=50 \
    --num_examples_per_epoch=298 --num_epochs=60 \
    --hparams=s3://wsum/lunalewa/WSUM-4347/DrOetker_BE_automl_fine_tune_WSUM-4739_train_2021-10_20_08_57_45_c9141_dd_24112021/config_lr_0_01/config.yaml`

config.yaml contains:
`num_classes: 226
autoaugment_policy: v2
max_instances_per_image: 300
aspect_ratios: [1.0, 2.0, 0.5, 8.0, 0.125]
nms_configs:
    max_output_size: 300
map_freq: 1
grad_checkpoint: true
learning_rate: 0.01
lr_warmup_init: 0.008
label_map: {1: 'DroBisClaBagBol_250g_00', 2: 'DroBisClaBagBolSid_250g_00', .....}`

	

Second training starts from first training pretrained_ckpt.
`python /repo/automl-1.2/efficientdet/keras/train.py \
    --train_file_pattern=s3://wsum/kubeflow/DrOetker_BE_automl/Clone_of_Run_of_AutoML_train_2021-11-24-12:06:30_8edb2/2021-11-25T12:07:1800:00/data/tfrecords/train.tfrecord \
    --val_file_pattern=s3://wsum/kubeflow/DrOetker_BE_automl/Clone_of_Run_of_AutoML_train_2021-11-24-12:06:30_8edb2/2021-11-25T12:07:1800:00/data/tfrecords/test.tfrecord \
    --model_name=efficientdet-d2 \
    --model_dir=s3://wsum/kubeflow/DrOetker_BE_automl/Clone_of_Run_of_AutoML_train_2021-11-24-12:06:30_8edb2/2021-11-25T12:07:1800:00/model_dir \
    --pretrained_ckpt=s3://wsum/lunalewa/WSUM-4347/DrOetker_BE_automl_fine_tune_WSUM-4739_train_2021-11-24-12_06_30_1d50c_dd_25112021/chckpt_60 \
    --batch_size=5 \
    --eval_samples=50 \
    --num_examples_per_epoch=298 --num_epochs=60 \
    --hparams=s3://wsum/lunalewa/WSUM-4347/DrOetker_BE_automl_fine_tune_WSUM-4739_train_2021-11-24-12_06_30_1d50c_dd_25112021/config_lr_0_004/config.yaml`

config.yaml contains:
`num_classes: 226
autoaugment_policy: v2
max_instances_per_image: 300
aspect_ratios: [1.0, 2.0, 0.5, 8.0, 0.125]
nms_configs:
    max_output_size: 300
map_freq: 1
grad_checkpoint: true
learning_rate: 0.004
lr_warmup_init: 0.008
label_map: {1: 'DroBisClaBagBol_250g_00', 2: 'DroBisClaBagBolSid_250g_00', .....}`

epoch_learning_rate tensorboard plot:
![image](https://user-images.githubusercontent.com/13466394/144305676-752922b7-eb33-4bef-a9ed-880ff2984c21.png)

I didn't specify lr_decay_method in both trainigns so it was 'cosine' by default.

Question:

How to setup 'cosine' learning rate schedule so that training would start with the learning_rate value at the top of cosine wave and deacrease to 0 at the end of training? 
In the same way as 
learning_rate: {
        cosine_decay_learning_rate {
          learning_rate_base: 0.005
          total_steps: 115200
          warmup_learning_rate: 0.0
          warmup_steps: 5000
        }
      }

from  TensorFlow Object Detection API does.
![image](https://user-images.githubusercontent.com/13466394/144305232-1fdf3100-c9bf-40e0-8f38-d3e7a069be1e.png)
